CHANGELOG.md

```diff
@@ -1,5 +1,44 @@
+* **New Resource:** `google_netapp_active_directory` ([#16844](https://github.com/hashicorp/terraform-provider-google/pull/16844))
+* **New Resource:** `google_vertex_ai_feature_online_store` ([#16840](https://github.com/hashicorp/terraform-provider-google/pull/16840))
+* **New Resource:** `google_vertexai_feature_group` ([#16842](https://github.com/hashicorp/terraform-provider-google/pull/16842))
+* **New Resource:** `google_netapp_backup_vault` ([#16876](https://github.com/hashicorp/terraform-provider-google/pull/16876))
+
+IMPROVEMENTS:
+* bigqueryanalyticshub: added `restricted_export_config` field to `google_bigquery_analytics_hub_listing ` resource ([#16850](https://github.com/hashicorp/terraform-provider-google/pull/16850))
+* composer: added support for `composer_internal_ipv4_cidr_block` field to `google_composer_environment` ([#16815](https://github.com/hashicorp/terraform-provider-google/pull/16815))
+* compute: added `provisioned_iops`and `provisioned_throughput` fields under `boot_disk.initialize_params` to `google_compute_instance` resource ([#16871](https://github.com/hashicorp/terraform-provider-google/pull/16871))
+* compute: added `resource_manager_tags` and `disk.resource_manager_tags` for `google_compute_instance_template` ([#16889](https://github.com/hashicorp/terraform-provider-google/pull/16889))
+* compute: added `resource_manager_tags` and `disk.resource_manager_tags` for `google_compute_region_instance_template` ([#16889](https://github.com/hashicorp/terraform-provider-google/pull/16889))
+* dataproc: added `auxiliary_node_groups` field to `google_dataproc_cluster` resource ([#16798](https://github.com/hashicorp/terraform-provider-google/pull/16798))
+* edgecontainer: increased default timeout on `google_edgecontainer_cluster`, `google_edgecontainer_node_pool` to 480m from 60m ([#16886](https://github.com/hashicorp/terraform-provider-google/pull/16886))
+* gkehub2: added field `version` under `configmanagement` in `google_gke_hub_feature` resource ([#16818](https://github.com/hashicorp/terraform-provider-google/pull/16818))
+* kms: added output-only field `primary` to `google_kms_crypto_key` ([#16845](https://github.com/hashicorp/terraform-provider-google/pull/16845))
+* metastore: added `endpoint_protocol`, `metadata_integration`, and `auxiliary_versions` to google_dataproc_metastore_service ([#16823](https://github.com/hashicorp/terraform-provider-google/pull/16823))
+* sql: added support for IAM GROUP authentication in the `type` field of `google_sql_user` ([#16853](https://github.com/hashicorp/terraform-provider-google/pull/16853))
+* storagetransfer: made `name` field to be optional in resource `google_storage_transfer_job` so it can be provided by the users ([#16838](https://github.com/hashicorp/terraform-provider-google/pull/16838))
+* vertexai: added `force_destroy` field to `google_vertex_ai_feature_online_store` resource ([#16857](https://github.com/hashicorp/terraform-provider-google/pull/16857))
```

Comment:
`google_vertex_ai_feature_online_store` was added in this release so I think we can leave this off.

```suggestion
```